i18n/ind/docusaurus-plugin-content-docs/current/lisk-tools/oracles.md

```diff
@@ -1,42 +1,44 @@
 ---
-title: Oracles
-slug: /lisk-tools/oracles
-description: Documentation for various blockchain oracles for Lisk.
-keywords:
-  [
-    Oracles,
-    Oracle,
-    Lisk,
-    Lisk Mainnet,
-    Lisk Testnet,
-    Lisk network,
-    Redstone,
-    price feeds,
-    data feeds,
-  ]
+
+title: Oracle  
+slug: /lisk-tools/oracles  
+description: Dokumentasi untuk berbagai orakel blockchain untuk Lisk.  
+keywords:  
+  [  
+    "Oracle",  
+    "Oracle",  
+    "Lisk",  
+    "Lisk Mainnet",  
+    "Lisk Testnet",  
+    "jaringan Lisk",  
+    "Redstone",  
+    "umpan harga",  
```

Comment:
I find no reference that price / data feed is translated as umpan harga / data at all in any article on the internet. The literal translation of umpan is "bait" / "lure". So it's better to retain the English wording for feed here.
```suggestion
    "feed harga",  
```